src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/services/impl/BatchedReconciliationService.kt

```diff
@@ -24,11 +28,26 @@ class BatchedReconciliationService(
 
 
     private fun unreconciledRecords(): List<UnreconciledRecord> = runBlocking {
-        metadataStoreRepository.groupedUnreconciledRecords(minimumAgeScale, minimumAgeUnit)
-            .map { (topic, records) ->
-                    async(Dispatchers.IO) {
-                        hbaseRepository.recordsInHbase(topic, records)
-                    }
+        val rdbmsTimedValue = measureTimedValue {
+            metadataStoreRepository.groupedUnreconciledRecords(minimumAgeScale, minimumAgeUnit);
+        }
+
+        logger.info("Queried rdbms", "duration" to "${rdbmsTimedValue.duration}")
```

Comment:
This naming isn't like the rest, that call it's metadatastore

Reply:
I've updated it - can you re-review?